<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFREAK</class>
 <widget class="QDialog" name="DialogFREAK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1" colspan="2">
    <widget class="QCheckBox" name="cBOrientationNormalized">
     <property name="text">
      <string>orientation normalized</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QCheckBox" name="cBScaleNormalized">
     <property name="text">
      <string>scale normalized</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>patternscale</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDoubleSpinBox" name="sBPatternScale">
     <property name="value">
      <double>22.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="2">
    <widget class="QCheckBox" name="cBDrawKeyPoints">
     <property name="text">
      <string>draw keypoints</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>noctaves</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QSlider" name="sliderNOctaves">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1" rowspan="5" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>98</width>
       <height>151</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QCheckBox" name="cBFASTKeyPoints">
     <property name="text">
      <string>fast keypoints</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QCheckBox" name="cBORBKeyPoints">
     <property name="text">
      <string>orb keypoints</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QCheckBox" name="cBBRISKKeyPoints">
     <property name="text">
      <string>brisk keypoints</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="15">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
